tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
+
+import pytest
+
+
+def pytest_addoption(parser: pytest.Parser):
+    # --openstack-rc points to an openstack credential file in the "rc" file style
+    # Here's an example of that file
+    # $ echo ~/openrc
+    # export OS_REGION_NAME=RegionOne
+    # export OS_PROJECT_DOMAIN_ID=default
+    # export OS_AUTH_URL=http://10.0.0.1/identity
+    # export OS_TENANT_NAME=demo
+    # export OS_USER_DOMAIN_ID=default
+    # export OS_USERNAME=demo
+    # export OS_VOLUME_API_VERSION=3
+    # export OS_AUTH_TYPE=password
+    # export OS_PROJECT_NAME=demo
+    # export OS_PASSWORD=nomoresecret
+    # export OS_IDENTITY_API_VERSION=3
+    parser.addoption("--openstack-rc", action="store", default="")
```

Comment:
Is there any impact to not having this parameter? Same question for `--content-cache-image`

Reply:
without this parameter you can't connect to your local devstack instance so you cannot perform operations in the swift storage. without the content-cache-image you wont be able to deploy the content-cache-k8s charm, since it needs a docker image a s a resource.